xlgui/widgets/playlist.py

```diff
@@ -734,7 +734,9 @@ def __init__(self, playlist, player):
         self.selection.set_mode(Gtk.SelectionMode.MULTIPLE)
 
         self._filter_matcher = None
-        
+
+        self._sort_columns = [] # Column sort order
```

Comment:
Seems like it would be easier to just initialize this to `list(common.BASE_SORT_TAGS)`?

Reply:
That might indeed be a better idea.